hudi-integ-test/src/main/java/org/apache/hudi/integ/testsuite/HoodieDeltaStreamerWrapper.java

```diff
@@ -53,14 +54,37 @@ public HoodieDeltaStreamerWrapper(Config cfg, JavaSparkContext jssc) throws Exce
     return upsert(WriteOperationType.BULK_INSERT);
   }
 
+  public JavaRDD<WriteStatus> insertOverwrite() throws
+          Exception {
+    return insertOverwrite(WriteOperationType.INSERT_OVERWRITE);
+  }
+
+  public JavaRDD<WriteStatus> insertOverwrite(WriteOperationType operation) throws Exception {
+    cfg.operation = operation;
+    return deltaSyncService.get().getDeltaSync().syncOnce().getRight();
+  }
+
+  public JavaRDD<WriteStatus> insertOverwriteTable() throws
+          Exception {
+    return insertOverwriteTable(WriteOperationType.INSERT_OVERWRITE_TABLE);
+  }
+
+  public JavaRDD<WriteStatus> insertOverwriteTable(WriteOperationType operation) throws
```

Comment:
Instead of new method, can we not reuse `insertOverwrite()` method in line 62? Note that these wrapper methods are only used for integration test suite.